typescript/test/baseline.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as assert from 'assert';
+import {execSync} from 'child_process';
+import * as fs from 'fs';
+import {ncp} from 'ncp';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+import * as util from 'util';
+
+const cwd = process.cwd();
+const rmrf = util.promisify(rimraf);
+const ncpp = util.promisify(ncp);
```

Comment:
given . the way we're using this in tests, I . don't think there's a . need to make this async.  Just use the sync methods?